datafusion/core/Cargo.toml

```diff
@@ -47,6 +47,7 @@ force_hash_collisions = []
 # Used to enable JIT code generation
 jit = ["datafusion-jit", "datafusion-row/jit"]
 pyarrow = ["pyo3", "arrow/pyarrow", "datafusion-common/pyarrow"]
+compression = ["xz2", "bzip2", "flate2", "async-compression"]
```

Comment:
I wonder how this is tested? 

The default features do not include compression

> default = ["crypto_expressions", "regex_expressions", "unicode_expressions"]

So that would imply to me we would need a change in `.github` to run a test with the feature flag... But I don't see any tests that fail 🤔

Reply:
i wonder if it makes sense to include this feature into default feature?

Reply:
without it, no tests will be run i guess...

Reply:
making it part of the default features makes sense to me

Reply:
i decided not to include it into default feature and at the same time update the GitHub action build to include it in the tests